data/Page/tips.md

```diff
@@ -0,0 +1,110 @@
+# How to look (and sound) good in your talk
+
+[This PDF by Kubecon EU is very useful](https://events.linuxfoundation.org/wp-content/uploads/2020/06/KubeCon_EU_Virtual_2020_WebPresenterDoc_v4.pdf), recommend reading.
+
+If you decide to buy additional hardware (note COVID is making stock levels challenging, order early), please make sure you put the time in to properly test before the event.  It’s unfortunately common to see someone buy an awesome new microphone to have them accidentally not have it set as the active recording audio device during their talk, or not have it placed in front of them correctly.
+
+## Microphone / audio quality
+
+By far, the best investment is ensuring audio quality is clear and not echo-y.  **This is more important than video quality.** Using your laptop microphone is OK for short talks or video chat, but the echo will get very fatiguing to listen to for a 20+ minute talk.
+
+In an ideal world; a external USB condenser microphone is your best bet, but you will need to practice with it to ensure you’re getting optimal audio levels, it’s positioned correctly, it doesn’t accidentally pickup your speakers/other sound during your talk, etc.
+
+Many podcasting guides are very useful to read, for example: https://www.buzzsprout.com/blog/mic-technique-podcasting
+
+Dropping down from this, using a USB headset with a microphone is the next best thing.
+
+Practice by recording some audio in something like Audacity, and listen to the difference in audio quality between your laptop microphone or the positioning of the microphone.  Pay attention to the audio control panel in your OS to ensure you are positioned in front of the microphone correctly (if you are using an analog microphone).
+
+When presenting, try to isolate yourself away from any background/ambient noise (close windows and doors, away from people if possible, etc).
+
+## Cameras & lighting
+
+Some laptop webcams work pretty well (i.e. those in a MacBook); others aren’t so good.  An external USB webcam such as a Logitech C920 is a relatively affordable way to add high quality video.
+
+That said, before you do that, make sure you’re in a well lit position - it’s surprising how much difference this makes to quality.  Lighting should be in front of you (facing back towards you), with no visible lights behind you.
+
+An easy/cheap way to ensure you’re well lit is to sit facing a window: but keep in mind the time of day when you test, compared to the time of day when you are presenting.
+
+## Camera positioning
+
+Make sure you are filling the frame - you don’t want your head at the bottom of the shot with lots of empty space!
```

Comment:
Richard suggested I provide a visual example, which is a good idea.

![image](https://user-images.githubusercontent.com/138684/88631241-3734e600-d0f5-11ea-83e4-9e3d9aec09af.png)